days/001-004-flask-intro/code/program/routes.py

```diff
@@ -4,7 +4,7 @@
 @app.route('/')
 @app.route('/index')
 def index():
-    return render_template('index.html', title='Template Demo', time=timenow)
+    return render_template('index.html', title='Template Demo')
```

Comment:
Great catch, thanks @serixscorpio.

Rather than deleting the variable could you actually set the time (`datetime.now()` I guess), because I see this in the `program/templates/index.html` template when I fork the repo: `<h1>Hello! The current date is {{ time }}</h1>` (and the related video might show the same). Thanks.

Reply:
Oops, I didn't notice `time` is used. Fixed in 511d080

Reply:
Thanks